Library/Homebrew/cmd/certs.rb

```diff
@@ -0,0 +1,50 @@
+#:  * `certs` [`--force-curl`]:
+#:    You should really write more documentation up here Dom.
+
+module Homebrew
+  BREWED_CURL = Pathname.new(HOMEBREW_PREFIX/"opt/curl")
+  BREWED_OPENSSL = Pathname.new(HOMEBREW_PREFIX/"opt/openssl")
+  BREWED_LIBRESSL = Pathname.new(HOMEBREW_PREFIX/"opt/libressl")
+  BREWED_GNUTLS = Pathname.new(HOMEBREW_PREFIX/"opt/gnutls")
+
```

Comment:
That's fair. But since you _do_ "police" formula versions, it might be nice to have a list for formula that install certs, and make sure their update PRs don't languish. Or something. Just a suggestion.